fe/fe-core/src/main/java/com/starrocks/memory/ProcProfileCollector.java

```diff
@@ -51,7 +51,7 @@ public class ProcProfileCollector extends FrontendDaemon {
     private long lastLogTime = -1;
 
     public ProcProfileCollector() {
-        super("ProcProfileCollector");
+        super("ProcProfileCollector", 1000L);
```

Comment:
would it cause performance issue?

Reply:
No performance issue, it's the check inverval, The profile interval is controlled by proc_profile_collect_interval_s